newsletter-examples/NOAA-CRW-DHW.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Extraction of a time series of Degree Heating Weeks (DHW) from the NOAA CRW collection\n",
+    "\n",
+    "*This is the notebook used to create the timeseries plot showcased in the DMS Newsletter #2*\n",
+    "\n",
+    "If you want to try this notebook using NECTAR's JupyterHub service just click here (you need to have an academic email registered with AAF): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ausseabed/dms-newsletter-2/HEAD?filepath=notebooks%2FDHW_timeseries.ipynb)\n",
+    "\n",
+    "What you need to know:\n",
+    "- the location of the Great Barrier Reef you want to assess. It must be a single point (lat, lon)\n",
+    "- the time period you want to assess. It must be a range of years (e.g. 2010-2019)\n",
+    "- the name of the output file you want to create. It must be a string (e.g. 'my_output_file.csv')\n",
+    "- the URI (Uniform Resource Identifier) of the NOAA CRW collection\n",
+    "\n",
+    "In this example we will use the DHW zarr collection managed by the DMS\n",
+    "\n",
+    "First we need to load the libraries we will use:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "import pandas as pd\n",
+    "# Accessing S3 buckets\n",
+    "import s3fs\n",
+    "from matplotlib import pyplot as plt\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-11-12T22:43:59.923346709Z",
+     "start_time": "2023-11-12T22:43:59.913888943Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We want to extract a time series of DHW values for *Mymidon Reef* (lat: -18.5, lon: 147.5) for the period 1995-2023 and save it in a csv file called 'my_output_file.csv'\n",
+    "\n",
+    "Let's define these variables:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "latitude = -18.5\n",
+    "longitude = 147.5\n",
+    "time_range = ['1995', '2023']\n",
+    "output_file = 'my_output_file.csv'\n",
```

Comment:
There is no use for output_file as well.

Reply:
it will be used at the end